examples/src/main/java/com/example/ftclibexamples/Odometry/DeadWheelsSample.java

```diff
@@ -63,6 +63,10 @@ public void runOpMode() throws InterruptedException {
         rightOdometer = frontRight.encoder.setDistancePerPulse(DISTANCE_PER_PULSE);
         centerOdometer = backLeft.encoder.setDistancePerPulse(DISTANCE_PER_PULSE);
 
+        // We reverse this because of the direction it is moving
+        // similarly to how we reverse the right side of the mecanum drive
+        rightOdometer.setDirection(Motor.Direction.REVERSE);
+
```

Comment:
we should draw an ascii map of the orientation and directions of the wheels

Reply:
just copy the RR quickstart :)